### DISCUSSION TITLE
[RFC] Change `blitz start` to `blitz dev` and `blitz start --production` to `blitz start`

### DISCUSSION THREAD

**Discussion by flybayer** [blitz-js/blitz#1869] [RFCs] [answered]
## Proposal

- Change dev start command from `blitz start` to `blitz dev`
- Change production start command from `blitz start --production` to `blitz start`

## Background

I initially added the run commands as `blitz start` and `blitz start --production` which matches Rails. But I'm seeing the writing on the wall that the JS community is moving towards `X dev` for starting the development server.

**Next.js**

```
Development:  next dev
Production:   next start
```

**Redwood.js**

```
Development:  rw dev
Production:   ?
```

**Twitter**

[![image](https://user-images.githubusercontent.com/8813276/106959888-526a5900-6709-11eb-86fa-e8145e9b5225.png)](https://twitter.com/jaredpalmer/status/1353083349800267776)

## Requesting Your Comments!

What do you think?

**Comment by robertt** [comment]
Makes sense - this is how Next does it.

**Comment by jan-wilhelm** [comment]
I personally find `start` to start the development server a lot more intuitive. As a user I simply want to literally "start" the server - I would not have a train of thought that goes "oh I want to _develop_ that's why I'll type _dev_"). Just my two cents, it's not that hugely importantt though :D

**Comment by paulm17** [comment]
So I'm thinking about intuitiveness and how it would work aloud.

**blitz dev** - I'm developing something, so start the local server in dev mode
**blitz build** - I've stopped developing, let me build it to put it on a prod server
**blitz start** - I'm on the prod server, lets start serving requests

Nice and simple and even new users can understand the logic.

**Comment by flybayer** [comment, marked as answer]
Looks pretty clear that most are in agreement to do this :)

PR opened: https://github.com/blitz-js/blitz/pull/1872
